packages/manager/modules/vrack/src/dashboard/vrack.controller.js

```diff
@@ -171,6 +173,13 @@ export default class VrackMoveDialogCtrl {
         this.$timeout.cancel(this.poller);
       }
     });
+
+    this.showModal = () => {
```

Comment:
Can you check if it is possible to use moveDialog component that already exist on vrack ? If it is not possible, try to not use jquery but use a component for the modal and use n-gi to display it or not like it is done here : https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/account/identity-documents/upload-identity-documents-confirm/template.html